### ISSUE TITLE
New Issue With update. System actions ghosted

### ISSUE THREAD

**Issue by Adek485** [Heckie75/kodi-addon-timers#19]
Kodi 19.3 on Nvidia Shield Pro

With previous timer version (I believe 3.0.0) and unknown Kodi sub-version within v19 I had a timer set to quit kodi at 3am. About May/June it stopped working. I rolled back timers to 3.0.0 and it still wouldn't quit Kodi according to the timer set. 
3.2.0 the option is unable to be selected. I looked at the code, but I got lost really early. I feel like it should be in settings_utils or system_utils but didn't see anything there. 

Looking to have the system action section enabled so I can have kodi quit at a given time (not a sleep timer)

**Comment by Heckie75** [comment]
You have written that '3.2.0 the option is unable to be selected'. I guess that the reason is that settings must be set to 'expert mode/level'. This is in the settings dialog, see wheel icon at the left corner. 

Have you tried this?

**Comment by Heckie75** [comment]
If this won't work, pls. send me the "timers.json" file where the timers are stored, see ./userdata/addon_data/script.timers/timers.json

**Comment by Adek485** [comment]
I can see the option, but can't select it.  The fading option is ghosted too.
I'll see about sending the timers file later

**Comment by Adek485** [comment]
My timers.json file is empty because I don't have any timers, because I can't select the option I want

**Comment by Heckie75** [comment]
@Adek485  

Ok, this screen does not show the relevant section. The precondition in order to select a system action is that the timer has as end, e.g. 'specific time' or 'duration', since the system action is performed after the timer has run out. 

If this doesn't help please send me the settings.xml file which is located in ./userdata/addon_data/script.timers/

Maybe it would also be a solution to delete this file.

**Comment by Heckie75** [comment]
@Adek485 
Thanks for correcting the spelling errors.

You have configured to 'shutdown Kodi'. This performs the default action that is configured in system settings (system > power saving > shutdown function) (expert mode only).

Have you tried other system actions e.g. suspend. I have tested the time addon on Linux und Windows systems and it works like a charme.

**Comment by Adek485** [comment]
Like I said I'm on Nvidia Shield Pro, which utilizes Android OS (linux flavor) and the only options I have are quit and minimize.  I had quit selected, so for troubleshooting purposes I elected minimize, and there was no action that was taken.

Changed it back to quit and tested the system function. Where the screenshot says "off" the only options are countdown timers from 120min -> 5 min in 5 min increments. I picked 5 min and walked away and it performed as I'd expect. 

![20221004_111507](https://user-images.githubusercontent.com/114942942/193894953-fc281434-0be7-4b73-938c-85a287551daa.jpg)

**Comment by Adek485** [comment]
Playing around with it, I think I found magic keywords. Instead of doing an end time, I made it a duration of 1 min. I recieved a pop up at that time saying something to the effect of "about to perform this action" then it quit a moment later. 

I turned off (hibernate/suspended) the shield, as we've done in the past, when it would quit successfully, and now it is working. Thank you for bearing with me in troubleshooting.
